src/NftDetails/NFTDetails.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import {storiesOf} from '@storybook/react-native'
```

Comment:
NftDetails is a use case of feature /nfts, please move it inside, also please normalize to Nft names